projects/storefrontstyles/scss/theme/sparta/_fonts.scss

```diff
@@ -1 +1 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap');
```

Comment:
I'm not sure if we can set it up like this. 

In my opinion as we want to have it released in minor proposed solution may be a breaking change by forcing existing customers after update to use `swap` strategy by default.

Instead, my proposal is maybe to expose new variable which default value will be `auto`, but `swap` can be set before Spartacus SCSS imports. Example: https://stackoverflow.com/a/59128372/13845333

The best option will be to ask Mikel (@morganm58) about it before.

Reply:
After looking into this. This is the new default when you copy the code from Google fonts now. `$font-url: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap';` looks correct to me.